requirements.txt

```diff
@@ -10,7 +10,7 @@ certifi==2016.2.28        # via tornado
 contextlib2==0.5.1        # via opentracing-instrumentation
 crcmod==1.7
 futures==3.0.5            # via opentracing, opentracing-instrumentation
-opentracing-instrumentation==2.2.0
```

Comment:
was this changed via `make requirements.txt`?

We should make sure that works since I beleive that's the code that runs the setup.py stuff.

Reply:
I believe I did that -- running `make requirements.txt` again just to be sure